### ISSUE TITLE
Model test of CMC is ineffective

### ISSUE THREAD

**Issue by seuzxy** [Deep-Learning-Person-Re-Identification/Implementaion-1#45]
The loss I trained on the CHUK03 data set is:
loss: 0.0057 - acc: 0.9997 - val_loss: 0.0048 - val_acc: 1.0000
Ｉtrained 5500 epoch.
the result of CMC is not good,rank1 to rank 20 are listed as below.
0.0
0.04
0.03
0.07
0.11
0.09
0.11
0.19
0.19
0.26
0.28
0.27
0.28
0.35
0.32
0.43
0.42
0.45
0.47
0.5
 I don't know why the effect is so bad, whether it's a training problem or a model problem. I use the CMC function in main.py.

**Comment by Joeykie** [comment]
i have the same question i guess may be __C.DATA.PATTERN.TRAIN = [1, 0, 0] in main.py should change to [1,1,0] to increase the amount of positive pair.

**Comment by Fawzikhattar** [comment]
Did u figure out what s wrong?
